MultiDOFJointValue[] values
duration time_from_start
